Add troubleshooting information for the 'authentication' step in HomeWizard (#37035)
home-assistant/home-assistant.github.io: source/_integrations/homewizard.markdown

- Make sure you have enabled the local API in device settings via the HomeWizard Energy app.
- Make sure both Home Assistant and the device are on the same network.

### Which button do I need to press to configure the device?

During setup, you may be asked to press a button on your device to authenticate it with Home Assistant.  
- **P1 Meter**: Press the white button on the front of the P1 Meter.  
- **Plug-In Battery**: Press the black touch button on the front of the device. You will hear a beep.  

The Water Meter and kWh Meter do not require this step.

> {% note %}

Reviewer: This is a note that is required for user to succeed in their task.

```suggestion
{% important %}
```

After pressing the button, you must select **Continue** within 30 seconds to complete the setup. If the setup times out, you may need to press the button again.

> {% endnote %}

Reviewer: ```suggestion
{% endimportant %}
```

> During setup, you may be asked to press a button on your device to authenticate it with Home Assistant.  
> - **P1 Meter**: Press the white button on the front of the P1 Meter.  
> - **Plug-In Battery**: Press the black touch button on the front of the device. You will hear a beep.  
> 
> The Water Meter and kWh Meter do not require this step.
> 
> {% important %}
> After pressing the button, you must select **Continue** within 30 seconds to complete the setup. If the setup times out, you may need to press the button again.
> {% endimportant %}

Reviewer 2: ```suggestion
1. During setup, you may be asked to press a button on your device to authenticate it with Home Assistant.  
- **P1 Meter**: Press the white button on the front of the P1 Meter.  
- **Plug-In Battery**: Press the black touch button on the front of the device. You will hear a beep. 
- **Water Meter** and **kWh Meter**: they do not require this step.
2. After pressing the button, you must select **Continue** within 30 seconds to complete the setup. 
    - If the setup times out, you may need to press the button again.
```
After re-reading this: As this is procedural content, I think ordered steps would make sense here.

Author: Sure, but keep in mind that all instructions are provided by the config flow.

Reviewer 2: right, thank you for the reminder! but still. I think it's better to keep procedural content in a procedural style in the docs.

- Make sure you have enabled the local API in device settings via the HomeWizard Energy app.
- Make sure both Home Assistant and the device are on the same network.

### Which button do I need to press to configure the device?

During setup, you may be asked to press a button on your device to authenticate it with Home Assistant.  
- **P1 Meter**: Press the white button on the front of the P1 Meter.  
- **Plug-In Battery**: Press the black touch button on the front of the device. You will hear a beep.  

The Water Meter and kWh Meter do not require this step.

{% note %}

> After pressing the button, you must click 'Continue' within 30 seconds to complete the setup. You may need to press the button again if the setup times out.

Reviewer 2: ```suggestion
After pressing the button, you must select **Continue** within 30 seconds to complete the setup. If the setup times out, you may need to press the button again.
```

{% endnote %}

> 

Reviewer 2: ```suggestion
```
